src/DynamoCore/DynamoCore.csproj

```diff
@@ -293,6 +294,7 @@ limitations under the License.
       <LastGenOutput>Resources.en-US.Designer.cs</LastGenOutput>
       <SubType>Designer</SubType>
     </EmbeddedResource>
+    <EmbeddedResource Include="ProtoGeometryImages.resx" />
```

Comment:
Sorry for not catching this earlier, @aosyatnik. `DynamoCore` was designed without geometry in mind, so it cannot know anything about geometries. Having this here in the project works, but it is incorrect. For now I think the best place to move this is to the `Libraries\GeometryUiWpf` project. Sorry for the late request!